sdk/formrecognizer/azure-ai-formrecognizer/azure/ai/formrecognizer/_form_base_client.py

```diff
@@ -79,3 +116,33 @@ def __init__(self, endpoint: str, credential: Union[AzureKeyCredential, TokenCre
         )
         self._deserialize = _get_deserialize(self._api_version)
         self._generated_models = self._client.models(self._api_version)
+
+    @distributed_trace
+    def send_request(self, request: HttpRequest, *, stream: bool = False, **kwargs) -> HttpResponse:
```

Comment:
Do we need to expose the `stream` param? I know our generated layer is a few versions behind the newer DPG designs so I think it would be good if we have a test where we set it to True to make sure everything is working as expected.